### DISCUSSION TITLE
Can not build simple-broker

### DISCUSSION THREAD

**Discussion by DatutaX** [hsaturn/TinyMqtt#93] [General]
i try to compile the simple-broker with Arduino ide 2.2.1 but get error.
 #include <vector>
          ^~~~~~~~
compilation terminated.
exit status 1
Compilation error: exit status 1

how to compile this ?
Regards

**Comment by hsaturn** [comment]
Converted to issue  https://github.com/hsaturn/TinyMqtt/issues/94
